src/WebUI/locales/en.yml

```diff
@@ -37,6 +37,26 @@ dateTime:
     short: min
     long: minute | minutes
 
+timeAgo:
```

Comment:
Did you know about https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Intl/RelativeTimeFormat ? Can it be used here?

Reply:
https://github.com/vueuse/vueuse/issues/3034
let's just wait a little while ;)